Author: Roberto C. Sanchez <roberto@connexer.com>
Description: A patch to allow recognition of Heimdal with the heimdal-multidev
package
--- trunk.orig/plugins/gssapi.c
+++ trunk/plugins/gssapi.c
@@ -86,7 +86,7 @@
 
 static const char * GSSAPI_BLANK_STRING = "";
 
-#ifndef HAVE_GSS_C_NT_HOSTBASED_SERVICE
+#if !defined(HAVE_GSS_C_NT_HOSTBASED_SERVICE) && !defined(GSS_C_NT_HOSTBASED_SERVICE)
 extern gss_OID gss_nt_service_name;
 #define GSS_C_NT_HOSTBASED_SERVICE gss_nt_service_name
 #endif
